packages/azure-container-services/images/test-images/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "azure-test-app",
+  "version": "1.0.0",
+  "description": "a azure test container",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/instana/nodejs.git"
+  },
+  "main": "server.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "Arya Mohanan <arya.mohanan@ibm.com>",
+  "license": "ISC",
+  "dependencies": {
+    "node-fetch": "^2.6.0"
```

Comment:
Suggestion: Use native fetch and only allow Node >= 18 to use for building the images.